doc/user/content/release-notes.md

```diff
@@ -46,6 +46,11 @@ Use relative links (/path/to/doc), not absolute links
 Wrap your release notes at the 80 character mark.
 {{< /comment >}}
 
+{{% version-header v0.9.2 %}}
+
+- Rename the metrics scraping interval parameter to
+  --metrics-scraping-interval (was --metrics-scraping-frequency).
+
```

Comment:
I don't think the flag made it into v0.9.1 so I think we're safe to just add a release note about its addition?

Reply:
Thanks, PR updates.